docs/reference/hooks.md

```diff
@@ -779,7 +779,6 @@ The `get_url`, `is_shown`, `get_context_data` and `render_html` methods all acce
 -   `page` - for `view` = `'edit'` or `'revisions_revert'`, the page being edited
 -   `parent_page` - for `view` = `'create'`, the parent page of the page being created
 -   `request` - the current request object
--   `user_page_permissions` - a `UserPagePermissionsProxy` object for the current user, to test permissions against
```

Comment:
It seems we're also missing the docs for the newly added `lock` and `locked_for_user` introduced in #8911. Not sure if we want to document those though, so I'm leaving it for now.